.github/workflows/lambda.yml

```diff
@@ -35,10 +44,14 @@ jobs:
       - run: yarn test:lambda:ci
       - uses: ./.github/actions/node/latest
       - run: yarn test:lambda:ci
+      - if: success() || failure()
```

Comment:
Doesn't this condition always apply?

Reply:
Yes, so that in the case of a test run failure, we still have logs outputted from the Test Agent (within the Test Agent Logs you can view debug logs of handled traces).